code/mission/missionmessage.cpp

```diff
@@ -316,7 +316,8 @@ int add_avi( char *avi_name )
 	strcpy_s( extra.name, avi_name );
 	strcpy_s( extra.anim_data.filename, avi_name);
 	extra.num = -1;
-	generic_anim_load(&extra.anim_data);
+	generic_anim_load(&extra.anim_data);   // load only to validate the anim
+	generic_anim_unload(&extra.anim_data); // unload to not waste bmpman slots
```

Comment:
But how is it validated? `generic_anim_load` has an error return value but it is never checked so no matter if the animation is invalid or not no warning will be displayed or am I missing something obvious here?